Framework/Reflectometry/test/CreateTransmissionWorkspaceAutoTest.h

```diff
@@ -11,14 +11,14 @@
 #include "MantidAPI/FrameworkManager.h"
 #include "MantidAPI/MatrixWorkspace.h"
 #include "MantidAPI/WorkspaceHistory.h"
-#include "MantidAlgorithms/CreateTransmissionWorkspaceAuto.h"
+#include "MantidReflectometry/CreateTransmissionWorkspaceAuto.h"
```

Comment:
This also has a v2, is v1 needed?

Reply:
I think this can probably be removed. I'll double check and if it's ok remove it :+1: